src/logic/config/model/networkConfig.ts

```diff
@@ -0,0 +1,70 @@
+import { ETHEREUM_NETWORK } from 'src/config/networks/network.d'
+
+import { NetworkSpecificConfiguration } from 'src/config'
+
+type GasPriceOracle = {
+  type: string
+  uri: string
+  gasParameter: string
+  gweiFactor: string
+}
+
+export type NetworkConfig = {
+  chainId: ETHEREUM_NETWORK
+  chainName: string
+  rpcUri: {
+    authentication: string
+    value: string
+  }
+  safeAppsRpcUri: {
+    authentication: string
+    value: string
+  }
+  blockExplorerUriTemplate: {
+    address: string
+    txHash: string
+  }
+  nativeCurrency: {
+    name: string
+    symbol: string
+    decimals: number
+    logoUri?: string
+  }
+  transactionService: string
+  theme: {
+    textColor: string
+    backgroundColor: string
+  }
+  gasPrice: GasPriceOracle
+  ensRegistryAddress: string
+  recommendedMasterCopyVersion: string
+}
+
+// We start from small to bigger. First having less values in storage
+export type NetworkState = {
+  chainId: ETHEREUM_NETWORK
+  chainName: string
+  nativeCurrency?: {
+    name: string
+    symbol: string
+    decimals: number
+    logoUri?: string
+  }
+  theme?: {
+    textColor: string
+    backgroundColor: string
+  }
+}
+
+export const makeNetworkConfigFromSpecificConfiguration = (config: NetworkSpecificConfiguration): NetworkState => {
```

Comment:
Maybe just `createNetworkConfig`?

Reply:
Because I was mixing a bit trying to make it ready to transition to the config service I wanted to make clear that this function would be currently used for the static configuration it depends on how many steps we migrate to the config-service later